src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,38 @@
+.container {
+  @include position-value;
+  position: absolute;
+}
```

Comment:
follow checklist 
<img width="960" alt="image" src="https://user-images.githubusercontent.com/102411130/233289681-9657c54d-8af4-40d1-b6d1-4e3895185312.png">

make parent element of clock a bem block and make clock a bem element
don't forget to creaet a separate file for created bem block